.gitattributes

```
@@ -4,3 +4,14 @@ tests/data/*-win.php eol=crlf
 
 # Don't show bundled libraries as source files in GitHub
 bundle/**/* linguist-vendored
+
+/.github export-ignore
+/tests export-ignore
+/features export-ignore
+/bin export-ignore
+/phpunit.xml.dist export-ignore
+/composer.json export-ignore
+/composer.lock export-ignore
+/utils export-ignore
+/.phpcs.xml.dist export-ignore
+/.editorconfig export-ignore
```

I think these are also valid to exclude, being only dev related files:

```suggestion
/.actrc export-ignore
/.editorconfig export-ignore
/.gitattributes export-ignore
/.gitignore export-ignore
/.mailmap export-ignore
/.behat.yml export-ignore
/phpcs.xml.dist export-ignore
/phpstan.neon.dist export-ignore
```